packages/block-editor/src/components/link-control/input-search.js

```diff
@@ -13,31 +13,31 @@ import { URLInput } from '../';
 
 const LinkControlInputSearch = ( {
 	value,
-    onChange,
-    onSelect,
-    renderSuggestions,
-    fetchSuggestions,
-    onReset,
-    onKeyDown,
-    onKeyPress,
+	onChange,
+	onSelect,
+	renderSuggestions,
+	fetchSuggestions,
+	onReset,
+	onKeyDown,
+	onKeyPress,
 } ) => {
 	return (
 		<form>
 			<URLInput
 				className="block-editor-link-control__search-input"
 				value={ value }
 				onChange={ onChange }
-				onKeyDown={ ( event, suggestion ) => {
+				__experimentalOnKeyDown={ ( event, suggestion ) => {
```

Comment:
I think @retrofox will be removing all key handlers but you will need to get him to review this.

Reply:
gone